addon/components/denali-text-area.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2020, Verizon Media
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import Component from '@glimmer/component';
+import { arg } from 'ember-arg-types';
+import { oneOf, string } from 'prop-types';
+
+const states = ['active', 'error'];
+
+export default class DenaliTextAreaComponent extends Component {
+  @arg(oneOf(states)) state;
+  @arg(string) errorMsg;
+
+  get activeClass() {
+    return this.state === 'active' ? 'is-active' : undefined;
+  }
+
+  get errorClass() {
+    if (this.state === 'error') {
+      if (this.errorMsg) {
```

Comment:
Can we make the state be independent of message? You can have the error styles without an error.

Reply:
Ok, yeah I can change that